.gitignore

```diff
@@ -1,2 +1,4 @@
 bin/
 cover.out
+.DS_Store
+.idea/
```

Comment:
Please note that I'm not a maintainer of this repository. Just someone following this repository.

Your PR might be accepted, but if I was maintaining this repository I wouldn't accept such PR

Take a look at a comment I made a few months ago about such changes

https://github.com/ccoVeille/go-safecast/pull/35#pullrequestreview-2367333706

TL;DR; don't do that, please consider configuring your local gitconfig with core.excludesFiles

Reply:
Absolutely make sense. Rolled back